common/djangoapps/adaptive_learning/migrations/0002_remove_adaptivelearningenabledflag_enable_adaptive_learning.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('adaptive_learning', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='adaptivelearningenabledflag',
+            name='enable_adaptive_learning',
+        ),
+    ]
```

Comment:
@macornwell Let's consolidate these migrations. We're only just adding the `AdaptiveLearningEnabledFlag` model, so there's no need to introduce multiple migrations related to it in this PR. If you remove [L22](https://github.com/open-craft/edx-platform/blob/ebf3b859dcb49734e6d58a3a98f286fdceb09465/common/djangoapps/adaptive_learning/migrations/0001_initial.py#L22) from `0001_initial.py`, it should be possible to delete this migration (`0002_remove_adaptivelearningenabledflag_enable_adaptive_learning.py`) and still get to the desired DB state when running migrations.